src/client/components/SentReviewsList/SentReviewsList.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SentReviewsList.styles.css';
+import { SentReviewsComponent } from '../SentReviewsComponent/SentReviewsComponent';
+
+export const SentReviewsList = ({ reviews }) => {
+  return (
+    <div className="sent-reviews-list">
+      <h2>Sent reviews</h2>
+      {reviews.map((cvReviews, index) => {
+        return (
+          // eslint-disable-next-line react/no-array-index-key
+          <div key={index} className="review-wrapper">
+            <SentReviewsComponent reviews={cvReviews} />
```

Comment:
@ember-b-Moss What is SentReviewsComponent. I actually could not get test in story book as there is no any SentReviewComponent. Can it be defined so SentreviewsList independently could get tested?

Reply:
Hey @hamedveloper , I am not sure what you are asking me? The objective of my story was to render a component made by @valmdz , called SentReviewsComponent. I had to made a sum component of her storybook data and display it together. Her component is there online on storybook. Maybe you misspelled it? Here you wrote me the name as SentReviewComponent but it is actually called SentReviewsComponent. So Review is in plural as Reviews :)